### DISCUSSION TITLE
Allow Screen Capture Disable

### DISCUSSION THREAD

**Discussion by savidudu** [keepassxreboot/keepassxc#9540] [Q&A]
Hello Team,

Is there anyway we can greyed out the Allow Screen Capture in the KeePassXC 2.7.5 application ?

We do not want this feature to be enabled for our staff.

Please suggest.

Thanks

**Comment by droidmonkey** [comment]
No they isn't and your staff could also just launch the program with `--allow-screencapture` to turn it on all the time.
